### PR TITLE
quick steps using a script to generate the docker compose file

### LINKED ISSUE #489: Create simple install/run script
Create a script that installs the node (docker or local) and runs it based on a config
EX: https://bitwarden.com/help/install-on-premise-linux/

### PR DESCRIPTION
**Pull request by md00ux**
Fixes #489

Changes proposed in this PR:
- a shell script designed to gather minimal end-user input and generate a basic Docker Compose file
- a guide on how to use the script, along with references to various useful docs

### CODE REVIEW

**Comment by mihaisc** [comment]
lgtm

File: deployment/ocean-node-quickstart.sh [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
....io/public"],"chainId":1,"network":"mainnet","chunkSize":100},"10":{"rpc":"https://mainnet.optimism.io","fallbackRPCs":["https://optimism-mainnet.public.blastapi.io","https://rpc.ankr.com/optimism","https://optimism-rpc.publicnode.com"],"chainId":10,"network":"optimism","chunkSize":100},"137":{"rpc":"https://polygon-rpc.com/","fallbackRPCs":["https://polygon-mainnet.public.blastapi.io","https://1rpc.io/matic","https://rpc.ankr.com/polygon"],"chainId":137,"network":"polygon","chunkSize":100},"23294":{"rpc":"https://sapphire.oasis.io","fallbackRPCs":["https://1rpc.io/oasis/sapphire"],"chainId":23294,"network":"sapphire","chunkSize":100},"23295":{"rpc":"https://testnet.sapphire.oasis.io","chainId":23295,"network":"sapphire-testnet","chunkSize":100},"11155111":{"rpc":"https://eth-sepolia.public.blastapi.io","fallbackRPCs":["https://1rpc.io/sepolia","https://eth-sepolia.g.alchemy.com/v2/demo"],"chainId":11155111,"network":"sepolia","chunkSize":100},"11155420":{"rpc":"https://sepolia.optimism.io","fallbackRPCs":["https://endpoints.omniatech.io/v1/op/sepolia/public","https://optimism-sepolia.blockpi.network/v1/rpc/public"],"chainId":11155420,"network":"optimism-sepolia","chunkSize":100}}'
```
Comment (mariacarmina) [inline deployment/ocean-node-quickstart.sh:140]: should we guide the user to export for himself/herself? like '$RPCS'. I understand that the user can modify this, but it seems a bit hard to read and to modify.

**Review by mariacarmina — COMMENTED** [review]
a small comment on my end.

File: deployment/ocean-node-quickstart.sh (hunk 1 above)
Comment (mihaisc) [reply deployment/ocean-node-quickstart.sh:140]: not at this point, at a later stage maybe. Now we need to merge as fast as possible

Comment (mariacarmina) [reply deployment/ocean-node-quickstart.sh:140]: ok, I'll approve

**Review by mariacarmina — APPROVED** [review]
lgtm

**Review by mihaisc — APPROVED** [review]

**Pull request merged**